baselines/getting_started/aws/aws_check_logging/cloudtrail.tf

```diff
@@ -1,20 +1,26 @@
 # CloudTrail Best Practices:
 # Related to AWS CIS 2.02 Ensure CloudTrail log file validation is enabled (Scored)
+# https://turbot.com/v5/mods/turbot/aws-cloudtrail/inspect#/policy/types/trailLogFileValidation
 resource "turbot_policy_setting" "aws_cloudtrail_trail_log_validation" {
+  count    = var.enable_cloudtrail_trail_log_validation ? 1 : 0
   resource = turbot_smart_folder.aws_logging.id
   type     = "tmod:@turbot/aws-cloudtrail#/policy/types/trailLogFileValidation"
   value    = "Check: Enabled"
 }
 
 # Trail Status Check
```

Comment:
Here, these need their full path, you can get that from `https://turbot.com/v5/mods/turbot/aws-cloudtrail/inspect#/policy/types/trailStatus` 

You can see examples in check S3